### PR TITLE
i18n(es): update `sidebar`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updates `sidebar` based on this commit dd64836af45f33df4a99ab864eabb91fc9b8e204

### CODE REVIEW

File: docs/src/content/docs/es/guides/sidebar.mdx [hunk 1]
```diff
@@ -295,10 +295,12 @@ La configuración anterior genera la siguiente barra lateral:
 
 ### Variantes de insignia
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/sidebar.mdx:296]: I think this is missing the heading change with the new "and custom styling" part.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the update :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and except for 1 missing change, this looks good to me.

**Comment by dreyfus92** [comment]
Nice catch and thank you @HiDeoo for your review 😁

**Review by HiDeoo — APPROVED** [review]
Thanks for the fix :raised_hands:

**Pull request merged**